### ISSUE TITLE
Enhancement request: One Shot One Kill (Tango One) - 'discard everything' option. 

### ISSUE THREAD

**Issue by mredgecase** [SotMSteamMods/CauldronMods#1061]
For situations where the player would want to discard Tango's entire hand, it would be nice to have a 'discard everything' option rather than to compel the player to click each card. 

No rush! :)

**Comment by wrhyme29** [comment]
I don't think this will end up worthwhile as by switching to custom text, we lose the counter of number of cards already discarded. I'll leave this open to explore more in detail later
